<ui version="4.0" >
 <class>CMakeSetupDialog</class>
 <widget class="QMainWindow" name="CMakeSetupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CMakeSetup</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="CMakeSetup.qrc" >:/Icons/CMakeSetupDialog.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Where is the source code:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="SourceDirectory" />
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="BrowseSourceDirectoryButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Where to build the binaries:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="BinaryDirectory" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="BrowseBinaryDirectoryButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3" >
        <widget class="QCMakeCacheView" name="CacheValues" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="configureButton" >
           <property name="text" >
            <string>Configure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generateButton" >
           <property name="text" >
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton" >
           <property name="text" >
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCMakeCacheView</class>
   <extends>QTableView</extends>
   <header>QCMakeCacheView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CMakeSetup.qrc" />
 </resources>
 <connections/>
</ui>
